book/error-monitoring/error-monitoring-ios.md

```diff
@@ -1,8 +1,8 @@
 
 # Error Monitoring plugin - iOS
 
-The iOS Error Monitoring plugin for Zapp is based on `ZPCrashlogsBaseProvider` implementing the `ZPCrashlogsPluginProtocol`.
-The `ZPCrashlogsPluginProtocol` goes through all of the functions for initializing an error monitoring provider.
+The iOS Error Monitoring plugin for Zapp implementing the `CrashlogsPluginProtocol`.
+The `CrashlogsPluginProtocol` goes through all of the functions for initializing an error monitoring provider.
 
```

Comment:
please add a line from which zapp sdk version this protocol is supported (i would put 13 and above i guess?)